corehq/apps/zapier/api/v0_5.py

```diff
@@ -26,8 +27,22 @@ def __init__(self, initial=None):
         self.label = initial.get('label', '')
         self.help_text = initial.get('help_text', '')
 
+"""
+class CustomActionField(object):
+
+    def __init__(self, initial = None):
+        initial = initial or {}
+        self.type = initial.get('type', '')
+        self.key = initial.get('key', '')
+        self.label = initial.get('label', '')
+        self.help_text = initial.get('help_text', '')
+        self.required = initial.get('required', '')
+"""
 
 class ZapierCustomFieldResource(Resource):
+    """
+    Base class for custom field resources in Zapier
```

Comment:
You might want to rename this `BaseZapierCustomFieldResource`, then you can remove this docstring as the name of the class will tell you everything you need to know.